lib/index.js

```diff
@@ -11,6 +12,7 @@ module.exports = {
   firstUniqueChar: require("./firstUniqueChar"),
   floor: require("./floor"),
   increment: require("./increment"),
+  indexOf: require('./indexOf'),
```

Comment:
Please us double quotes for all your imports. This will be added as an eslint rule in another issue.